File: MyAnimeListJava/src/main/java/com/example/myanimelistjava/models/enums/Type.java
package com.example.myanimelistjava.models.enums;

public enum Type {
    TV("TV"),
    MOVIE("Movie"),
    MUSIC("Music"),
    OVA("OVA"),
    ONA("ONA"),
    SPECIAL("Special");

    private final String value;

    Type(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: MyAnimeList/src/main/kotlin/com/example/myanimelist/models/enums/Type.kt
package com.example.myanimelist.models.enums

import javafx.collections.FXCollections
import javafx.collections.ObservableList

enum class Type(val value: String = "Unknown") {
    TV("TV"),
    MOVIE("Movie"),
    MUSIC("Music"),
    OVA("OVA"),
    ONA("ONA"),
    SPECIAL("Special");

    companion object {
        val sample: ObservableList<String> = FXCollections.observableArrayList(Type.values().map { it.value }.toList())
    }
}
